operating/build-time-network-policy-tools.adoc

```diff
@@ -0,0 +1,57 @@
+|`roxctl netpol generate`
+|Generates Kubernetes network policies by analyzing your project's YAML manifests in a specified directory. For more information, see xref:../operating/build-time-network-policy-tools.adoc#using-the-build-time-network-policy-generator_network-policy-tools[Using the build-time network policy generator].
+
+|`roxctl netpol connectivity map`
+|Lists the allowed connections between workloads in your project directory by examining the workload and Kubernetes network policy manifests. You can generate the output in various text formats or in a graphical `.dot` format. For more information, see xref:../operating/build-time-network-policy-tools.adoc#connectivity-mapping-using-the-roxctl-netpol-connectivity-map-command_network-policy-tools[Connectivity mapping using the roxctl netpol connectivity map command].
+
+|`roxctl netpol connectivity diff`
+|Creates a list of variations in the allowed connections between two project versions. This is determined by the workload and Kubernetes network policy manifests in each version's directory. This feature shows the semantic differences which are not obvious when performing a source code (syntactic) `diff`. For more information, see xref:../operating/build-time-network-policy-tools.adoc#identifying-the-differences-in-allowed-connections-between-project-versions_network-policy-tools[Identifying the differences in allowed connections between project versions].
+
+|===
+
+//Network policy generator
+[id="using-the-build-time-network-policy-generator_{context}"]
+== Using the build-time network policy generator
+
+:FeatureName: Build-time network policy generation
+include::snippets/technology-preview.adoc[]
+
+The build-time network policy generator can automatically generate Kubernetes network policies based on application YAML manifests. You can use it to develop network policies as part of the continuous integration/continuous deployment (CI/CD) pipeline before deploying applications on your cluster.
+
+Red Hat developed this feature in partnership with the developers of the link:https://np-guard.github.io/[NP-Guard project].
+First, the build-time network policy generator analyzes Kubernetes manifests in a local folder, including service manifests, config maps, and workload manifests such as `Pod`, `Deployment`, `ReplicaSet`, `Job`, `DaemonSet`, and `StatefulSet`.
+Then, it discovers the required connectivity and creates the Kubernetes network policies to achieve pod isolation.
+These policies allow no more and no less than the needed ingress and egress traffic.
+
+include::modules/generate-build-time-network-policies.adoc[leveloffset=+2]
+
+After generating the policies, you must inspect them for completeness and accuracy, in case any relevant network address was not specified as expected in the YAML files.
+Most importantly, verify that required connections are not blocked by the isolating policies. To help with this inspection you can use the `roxctl netpol connectivity map` tool.
```

Comment:
Could maybe add a `.Next steps` for this? Also missing comma near the end:
```suggestion
.Next steps

* After generating the policies, you must inspect them for completeness and accuracy, in case any relevant network address was not specified as expected in the YAML files.
* Most importantly, verify that required connections are not blocked by the isolating policies. To help with this inspection, you can use the `roxctl netpol connectivity map` tool.
```